t/unit-tests/t-reftable-pq.c

```diff
@@ -10,11 +10,11 @@ license that can be found in the LICENSE file or at
 #include "reftable/constants.h"
```

Comment:
C doesn't have references, so you should probably reword the commit message to say "pointer".